src/main/java/jenkins/plugins/collabnet/steps/PublishWebhookStep.java

```diff
@@ -218,6 +261,33 @@ private void log(String msg, PrintStream printStream) {
             printStream.print(LOG_MESSAGE_PREFIX);
             printStream.println(msg);
         }
+
+        public StandardUsernamePasswordCredentials getCredentials() {
+            return getCredentials(this.run.getParent(),
+                    this.step.getCredentialsId(), this.step.getWebhookUrl());
+        }
+
+        public static StandardUsernamePasswordCredentials getCredentials(Item owner,
+                                                                         String credentialsId, String webhookUrl) {
+            StandardUsernamePasswordCredentials result = null;
+            if (!isBlank(credentialsId)) {
+                for (StandardUsernamePasswordCredentials c : lookupCredentials(owner, webhookUrl)) {
+                    if (c.getId().equals(credentialsId)) {
+                        result = c;
+                        break;
+                    }
+                }
+            }
+            return result;
+        }
+
+        public static List<StandardUsernamePasswordCredentials> lookupCredentials(Item owner, String serverUrl) {
```

Comment:
You are using "PublishEventQStep.PublishEventQStepExecution.lookupCredentials"  in PublishWebhooksStep.java. You can use the same here if the implementation is the same OR you can use this method in PublishWebhooksStep.java to be consistent.

Reply:
Yes, lookupCredentials() same here in PublishWebhookStep.java . Do you want to move this methods to Helper.class? thought it would be easy to remove when deprecate EventQ.